inference-engine/src/transformations/include/transformations/rt_info/primitives_priority_attribute.hpp

```diff
@@ -14,15 +19,28 @@
 
 namespace ngraph {
 
+/**
+ * @ingroup ie_runtime_attr_api
+ * @brief PrimitivesPriority class represents runtime info attribute that
+ * can be used for plugins specific primitive choice.
+ */
 class TRANSFORMATIONS_API PrimitivesPriority {
 private:
     std::string primitives_priority;
 
 public:
+    /**
+     * A default constructor
+     */
     PrimitivesPriority() = default;
 
+    /**
+     * @brief      Constructs a new object consisting of a single name     *
+     * @param[in]  name  The primitives priority value
+     */
     explicit PrimitivesPriority(const std::string &primitives_priority) : primitives_priority(primitives_priority) {}
 
+    // return string with primitives priority value
```

Comment:
does doxygen understand comments in such form?

Reply:
all methods must be documented with brief description, arguments (if any), return (if non-void)

Reply:
Added `@brief` for this methods. But why should we specify `@return` if it's already parsed by doxygen?

Reply:
I think `@return` is not needed here.